tests/lax_numpy_operators_test.py

```diff
@@ -731,6 +731,29 @@ def testSpacingSubnormals(self, dtype):
     self._CheckAgainstNumpy(np.spacing, jnp.spacing, args_maker, check_dtypes=True, tol=0)
     self._CompileAndCheck(jnp.spacing, args_maker, tol=0)
 
+  def testFloorDivideByZeroInt(self):
+    x1 = jnp.eye(3, dtype=int)
+    x2 = jnp.zeros((3, 3), dtype=int)
+    jnp_result = jnp.floor_divide(x1, x2)
+    np_floor_divide = jtu.ignore_warning(category=RuntimeWarning,
+                               message="divide by zero.*")(np.floor_divide)
+    np_floor_divide = jtu.ignore_warning(category=RuntimeWarning,
+                               message="invalid value.*")(np_floor_divide)
+    np_result = np_floor_divide(x1, x2)
+    self.assertAllClose(jnp_result, np_result, check_dtypes=True)
+
+  def testFloorDivideByZeroFloat(self):
+    x1 = jnp.array([-2., 3., -4., 6., -0., 2., -4., 0., -2., 0., -0., 0.], dtype=float)
+    x2 = jnp.array([-3., -2., 3., -7., 1., -0., 0., -2., 1., 0., 0., -0.], dtype=float)
+    jnp_result = jnp.floor_divide(x1, x2)
+    np_floor_divide = jtu.ignore_warning(category=RuntimeWarning,
```

Comment:
It's a bit clearer I think to put these `ignore_warning` statements as decorators for the whole test function.

Reply:
Thanks for the suggestion! Included `ignore_warning` statements as decorators.